include/substrait-mlir/Dialect/Substrait/IR/SubstraitOps.td

```diff
@@ -526,7 +526,8 @@ def Substrait_ProjectOp : Substrait_RelOp<"project", [
   }];
 }
 
-def SetOpKind : I32EnumAttr<"SetOpKind", "The enum values correspond to those in the SetRel.SetOp message.",[I32EnumAttrCase<"unspecified", 0>,
+def SetOpKind : I32EnumAttr<"SetOpKind", [{The enum values correspond to those 
+in the SetRel.SetOp message.}],[I32EnumAttrCase<"unspecified", 0>,
```

Comment:
I'd prefer something like this:
```suggestion
def SetOpKind : I32EnumAttr<"SetOpKind",
  "The enum values correspond to those in the SetRel.SetOp message.", [
    I32EnumAttrCase<"unspecified", 0>,
    # Remaining cases indented by four spaces
```

Reply:
I just looked at the failures in CI. It could be that multiline strings aren't allowed in that case (but my diff also happens to solve that). I *think* you can also do something like that (in which case there is no line break in the defined string):

```tablegen
"first line"
"second line"
```